server/issue.go

```diff
@@ -28,43 +28,64 @@ func makePost(userId, channelId, message string) *model.Post {
 	}
 }
 
+func respondf(plugin *Plugin, status int, e error, mmuserID, jiraBotID, channelID, message string) (int, error) {
```

Comment:
Can we rename this to something that implies we are returning an error? Mainly because there are no paths that return `nil` for the error return value. Maybe something like `respondErr`?

Reply:
Worked on this.